domains/transfer/Transfer.API/API/Transfer/TransferAgreementProposalCleanup/TransferAgreementProposalCleanupService.cs

```diff
@@ -53,7 +53,7 @@ private async Task DeleteOldTransferAgreementProposals(DateTimeOffset olderThan,
         context.TransferAgreementProposals.RemoveRange(oldProposals);
 
         var activityLogEntries = oldProposals.Select(proposal => ActivityLogEntry.Create(Guid.Empty, ActivityLogEntry.ActorTypeEnum.System,
-            string.Empty, proposal.SenderCompanyTin, proposal.SenderCompanyName, ActivityLogEntry.EntityTypeEnum.TransferAgreementProposal,
+            string.Empty, proposal.SenderCompanyTin, proposal.SenderCompanyName, String.Empty, String.Empty, ActivityLogEntry.EntityTypeEnum.TransferAgreementProposal,
```

Comment:
Maybe we should have 2 Create methods.

One for creating with all fields and one creating only internal for an organisation.

Eg: CreateOwn(...)
CreateWithOtherOrganisation(...)

So we dont pass string.empty as parameters?

Reply:
Just an idea if it's not too annoying :)

Reply:
Great suggestion. The plan with this pr, was to apply the migrations only. I will handle the business logic in my next PR.